src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookTriggerImpl.java

```diff
@@ -136,7 +136,7 @@ boolean isWorkflowJob(@Nullable SCMTriggerItem triggerItem) {
      *
      * For pipeline job, this is invoked every time a build is run.
      *
-     * We can't always continue registration check if newInstance is false since during Jenkin startup, this is invoked
+     * We can't always continue registration check if createInstance is false since during Jenkin startup, this is invoked
```

Comment:
was this intentional? Looks like IDEA trying to be too smart. ;-)

Reply:
Yeah, IntelliJ super sensitive refactor. I'll fix it.